src/components/Dropdown/Dropdown.jsx

```diff
@@ -97,11 +97,7 @@ export const Dropdown = ({
   };
 
   return (
-    <div
-      className="rvt-dropdown"
-      data-rvt-dropdown="dropdownNavigation"
```

Comment:
Just removing the unnecessary data-rvt* attributes. That's the only change in the Dropdown component in this PR.